### ISSUE TITLE
Total

### ISSUE THREAD

**Issue by avolaid** [deg0nz/MMM-Fronius2#1]
Thank you for that nice module.

First, TOTAL is showing - kW instead of mW
Second, Total calculation is wrong. It shows 4,48 KW but it should be 16,13 mW
![Screenshot 2022-03-15 at 18 20 49](https://user-images.githubusercontent.com/48850030/158435123-1b7dc666-8c29-48a8-b472-1be68794fa44.png)
![IMG_4279](https://user-images.githubusercontent.com/48850030/158435168-7cef41a7-d845-4d4d-bd86-2077fdccfaff.jpeg)

**Comment by deg0nz** [comment]
Hi @avolaid 

thanks for reporting this issue. I digged into this and recognized, that my power converter also shows weird values for "Total". It shows only ~300Wh, which is also wrong for my system. 

Could you please help me with the following to verify?

Please print the output of the following command from a terminal in a linux of unix system:

``` bash
curl http://<IP_ADDRESS>/solar_api/v1/GetPowerFlowRealtimeData.fcgi
```
Please adjust the IP address of your power converter in the command. If the `curl` command is not installed on your raspberry, you can install it like so: `sudo apt install curl`

Since your PV system has way bigger values than mine, I just want to clarify that I don't have some number overflow error here. Or if your converter also delivers wrong values.

**Comment by avolaid** [comment]
Thank you for fast replay. Here is the output:
``` json
{
   "Body" : {
      "Data" : {
         "Inverters" : {
            "1" : {
               "DT" : 123,
               "E_Day" : 11562,
               "E_Total" : 4482.35546875,
               "E_Year" : 1140327.75,
               "P" : 0
            }
         },
         "Site" : {
            "E_Day" : 11562,
            "E_Total" : 4482.3555555555558,
            "E_Year" : 1140327.75,
            "Meter_Location" : "unknown",
            "Mode" : "produce-only",
            "P_Akku" : null,
            "P_Grid" : null,
            "P_Load" : null,
            "P_PV" : null,
            "rel_Autonomy" : null,
            "rel_SelfConsumption" : null
         },
         "Version" : "12"
      }
   },
   "Head" : {
      "RequestArguments" : {},
      "Status" : {
         "Code" : 0,
         "Reason" : "",
         "UserMessage" : ""
      },
      "Timestamp" : "2022-03-16T17:51:18+01:00"
   }
}
```

**Comment by deg0nz** [comment]
Okay, so there is the same issue with your converter. The API of the converter only delivers 4,48 kWh as you can see in the `E_Total` value. So the root cause of this problem is not the MagicMirror module.

I will dig into the Fronius manuals and write an email to Fronius if necessary to clarify these values. Maybe we misinterpret "total" somehow...?!

On the web interface of my converter it also shows the low value. So we can cancel out issues with the API.

**Comment by deg0nz** [comment]
Ok, it was an error in the power converter software. The MagicMirror module is doing everything right.

A fix was released with V3.21.4-1. 
Please see release notes: http://www3.fronius.com/datalogger_web/dmc/updates/3.21.4-1//changelog.txt

I just updated my converter and everything works fine. Please update your converter SW. 
I'm closing this issue now.

**Comment by avolaid** [comment]
It works. Thank you
